src/D8/MenuLinkContentTrait.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
please move all this into `MenuTrait` as these traits are grouped by functionality rather than Drupal structures.